src/main/java/com/ldtteam/structurize/blueprints/v1/BlueprintUtil.java

```diff
@@ -74,6 +75,10 @@ public static Blueprint createBlueprint(
         for (final BlockPos mutablePos : BlockPos.betweenClosed(pos, pos.offset(sizeX - 1, sizeY - 1, sizeZ - 1)))
         {
             BlockState state = world.getBlockState(mutablePos);
+            if (state.is(ModTags.BLUEPRINT_BLACKLIST))
```

Comment:
uhh what? what's the code path of this method? we definitely want to save what we scan fully, such blacklisting should take place during structure placing